### DISCUSSION TITLE
Is there any command to refresh/reload opened files in code-server?

### DISCUSSION THREAD

**Discussion by dclong** [coder/code-server#2944] [Q&A] [answered]
I found myself update files frequently from command line. For example, format code using command line. The changes on opened files from command lines are not automatically updated to the opened editors. I wonder whether there is any command to refresh/reload files? Or is there any configuration to make opened files auto refresh/reload on changes from other source (say, command line)?

**Comment by oxy** [comment, marked as answer]
code-server _should_ reload any files on change except when you have some unsaved changes; I think File -> Revert should otherwise work, though.

Another issue that you might run into is the lack of available watcher slots on a Linux host - to fix, I'd recommend using `sysctl -w fs.inotify.max_user_watches=N`, with N being something like 524288. Keep in mind that every watcher uses up to 160 bytes, IIRC, so adjust the number based on the total memory on your host.
